ui/src/server/db/schema.ts

```diff
@@ -17,32 +17,31 @@ import { type AdapterAccount } from "next-auth/adapters";
  *
  * @see https://orm.drizzle.team/docs/goodies#multi-project-schema
  */
-export const mysqlTable = mysqlTableCreator((name) => `rego-ui_${name}`);
+export const pgTable = pgTableCreator((name) => `rego-ui_${name}`);
 
-export const posts = mysqlTable(
+export const posts = pgTable(
   "post",
   {
-    id: bigint("id", { mode: "number" }).primaryKey().autoincrement(),
+    id: bigint("id", { mode: "number" }).primaryKey(),
```

Comment:
I think you can define a CUID auto generated value

Reply:
it's for the default post table.
it's a throwaway code anyway, I wouldn't put much into it